docs/building-examples.rst

```diff
@@ -46,9 +46,9 @@ Next, prepare the environment:
       .. code-block:: bash
          :name: conda-prep-env-eos-s3
 
-         export PATH="$F4PGA_INSTALL_DIR/$FPGA_FAM/quicklogic-arch-defs/bin:$PATH";
-         export F4PGA_BIN_DIR="$F4PGA_INSTALL_DIR/$FPGA_FAM/quicklogic-arch-defs/bin";
-         export F4PGA_SHARE_DIR="$F4PGA_INSTALL_DIR/$FPGA_FAM/quicklogic-arch-defs/share/symbiflow";
+         export PATH="$F4PGA_INSTALL_DIR/$FPGA_FAM/install/bin:$PATH";
+         export F4PGA_BIN_DIR="$F4PGA_INSTALL_DIR/$FPGA_FAM/install/bin";
```

Comment:
If F4PGA_BIN_DIR and F4PGA_SHARE_DIR are consistent with xc7, we might not need to define these envvars. These are only required in case they are located somewhere other than the expected default path.
Let's see after the arch-defs packages are generated and bumped here.

Reply:
Done in https://github.com/chipsalliance/f4pga/commit/df55ee75e4a1da978ffa5f7385bed47faf255661, https://github.com/chipsalliance/f4pga-examples/pull/338/commits/a46951037bfc40890bb4b960c5ae59842b11838b and https://github.com/chipsalliance/f4pga-examples/pull/338/commits/52bc71a0506cad647951a16fb214ea82934bed0a.